Lesson34/selcan/lesson34-app/src/components/Form/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import { Account } from "../Account";
+
+// TODO: Create a new Component that contains log in form, so that user can type username and password and click submit button to log in.
+// If user is logged in, show Account component.
+// If user is logged out, show Log In component.
+
+export default function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [loggedIn, setLoggedIn] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    if (username === "user" && password === "password") {
+      setLoggedIn(true);
+    } else {
+      alert("wrong username or password!");
```

Comment:
Instead of using alert function, you can display error message on the screen like this:
```
// add a new state to hold error message
  const [error, setError] = useState("");

...

  const handleSubmit = (e) => {
    e.preventDefault();

    if (username === "user" && password === "password") {
      setLoggedIn(true);
    } else {
      // Replace alert with setError
      setError("Wrong username or password!");
    }
  };
...
// show the error message under the form element. Make sure to wrap form and the <p > element holding error in a Fragment element
<>
        <form onSubmit={handleSubmit}>
          // content of the form element
        </form>
        {error ? <p>{error}</p> : <></>}
        </>
...
// rest of the code

```